content-src/experiments/containers.yaml

```diff
@@ -58,7 +58,7 @@ tour_steps:
     copy: Start a Container to label and organize a set of tabs.
   -
     image: /static/images/experiments/containers/tour/tour2.jpg
-    copy: Hover over the New Tab button to select a container.
+    copy: Long press on the New Tab button to select a container.
```

Comment:
The build is failing because you changed the meaning of this string without changing the L10n ID, so translations won't be updated to reflect the new meaning. More info about [how we do localization](https://github.com/mozilla/testpilot/blob/master/docs/content/reference.md#localization).

You'll want to add something like:

```yaml
copy_l10nsuffix: 'longpress'
```